source/MRMesh/MRLineObject.h

```diff
@@ -42,7 +42,12 @@ class MRMESH_CLASS LineObject : public ObjectLinesHolder, public FeatureObject
     /// updates xf to scale size
     MRMESH_API void setLength( float size );
     /// calculates line size from xf
-    MRMESH_API float getLength() const;
+    [[nodiscard]] MRMESH_API float getLength() const;
+
+    /// Returns the starting point, aka `center - dir * len/2`.
+    [[nodiscard]] MRMESH_API Vector3f getPointA() const;
+    /// Returns the finishing point, aka `center + dir * len/2`.
+    [[nodiscard]] MRMESH_API Vector3f getPointB() const;
```

Comment:
just suggestion to name it like `org` `dest`

Reply:
Agreed to keep for now.